packages/mds-db/schema.ts

```diff
@@ -9,8 +9,11 @@ const AUDIT_EVENTS_TABLE = 'audit_events'
 const POLICIES_TABLE = 'policies'
 const GEOGRAPHIES_TABLE = 'geographies'
 
+const IDENTITY_COLUMN = 'id'
+
 // agency
 const DEVICES_COLS = [
+  IDENTITY_COLUMN,
   'device_id',
```

Comment:
do we want constants for these other fields, since they are replicated?

Reply:
Yeah, I am going to tidy up the schema types to prevent typos and omissions.